src/components/Footer/Footer.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './footer.css';
```

Comment:
it would be nice to have css modules

https://github.com/css-modules/css-modules

Reply:
Implemented + It was really interesting.